### DISCUSSION TITLE
Auto Enable Use Office Open XML instead of ODF by default

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#2266] [Ideas] [answered]
If the desire is to make NC AIO a serious business tool, it is time to put away the things of a child and don the tools of universal business. Sorry ODF loving people. That isn't you.

**Comment by szaimen** [comment, marked as answer]
Hi, we'll use whatever default comes from the richdocuments app. So feel free to open a feature request for this if there shouldn't be one already here: https://github.com/nextcloud/richdocuments/issues

**Reply by FlattusBlastus** [reply]
I think that's unwise and acting against the interest of Nextcloud's enterprise efforts. Can we engage with Andy Scherzinger on this? No one ever says send me those odt docs, would ya?

**Reply by FlattusBlastus** [reply]
Also isnt that "Nextcloud Docs" now? You shouldn't have it both ways. If not, please give us Only Office back.
